server/neptune/workflows/activities/github.go

```diff
@@ -316,7 +317,22 @@ type ListModifiedFilesResponse struct {
 }
 
 func (a *githubActivities) GithubListModifiedFiles(ctx context.Context, request ListModifiedFilesRequest) (ListModifiedFilesResponse, error) {
-	// TODO: Use client.ListModifiedFiles(ctx, owner, repo, pullNumber) to list files modified in a PR
-	// internal.Repo object and internal.PullRequest has all the necessary fields to make this call
-	return ListModifiedFilesResponse{}, nil
+	files, err := a.Client.ListModifiedFiles(
+		internal.ContextWithInstallationToken(ctx, request.Repo.Credentials.InstallationToken),
+		request.Repo.Owner,
+		request.Repo.Name,
+		request.PullRequest.Number,
+	)
+	if err != nil {
+		return ListModifiedFilesResponse{}, errors.Wrap(err, "listing modified files in pr")
+	}
+
+	filepaths := []string{}
+	for _, file := range files {
+		filepaths = append(filepaths, file.GetFilename())
```

Comment:
is the filename enough here?

Reply:
Yes, the name is misleading. It returns the absolute file path.